tests/test_de.py

```diff
@@ -65,3 +65,23 @@ def test_ordinal_for_negative_numbers(self):
 
     def test_ordinal_for_floating_numbers(self):
         self.assertRaises(TypeError, num2words, 2.453, ordinal=True, lang='de')
+
+    def test_to_currency_for_some_numbers(self):
+        self.assertEqual(
+            num2words(190, lang='de', to='currency'),
+            "eins euro und neunzig cents"
```

Comment:
Looks wrong to me. 190 is "einhundertneunzig Euro null Cent" or "hundertneunzig Euro null Cent". Also, the "null Cent" sounds weird. FYI "ein Euro neunzig Cent" means 1.90 euro.